@ndlib/gatsby-theme-marble/src/assets/i18n/json/text.default.en.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
This is where default text for the site will go. Things like the text first time users see. Or the about IIIF information.